src/components/NavBar /NavBar.jsx

```diff
@@ -0,0 +1,102 @@
+import {
+  Box,
+  Button,
+  Drawer,
+  DrawerContent,
+  DrawerHeader,
+  DrawerOverlay,
+  IconButton,
+  useDisclosure,
+} from "@chakra-ui/react";
+import React from "react";
+import { BiBookBookmark as ShoppingCartIcon } from "react-icons/bi";
+import SectionSeperator from "./SectionSeperator";
+import ShoppingCartItem from "./ShoppingCartItem";
+
+const NavBar = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [shoppingCartItems] = React.useState([
```

Comment:
for this project, we don't really import react. we just destructure like `import {useState} from 'react'`. Something really minor, but please try to follow this style. So basically it shouldn't be `React.useState()` but should be `useState` only.